src/components/title.tsx

```diff
@@ -16,11 +16,16 @@ const titleSizes = {
   xxlarge: "72px",
 }
 
-const Title: React.SFC<TitleProps> = props => (
-  <div className={props.className}>
-    {props.children}
-  </div>
-)
+const Title: React.SFC<TitleProps> = props => {
+  const newProps: TitleProps = {...props}
```

Comment:
I def see this pattern in our components a lot, it sort of makes sense in order to allow passing options/props along to components.